Simplified/NYPLBugsnagLogs.swift

```diff
@@ -170,4 +170,20 @@ fileprivate let tabName = "Extra Data"
       report.addMetadata(metadata, toTabWithName: tabName)
     })
   }
+  
```

Comment:
Function explaining what this function does and maybe short one-liner about the parameters?